docker-compose.yaml

```diff
@@ -80,6 +80,7 @@ services:
       - ARNS_CACHE_TTL_SECONDS=${ARNS_CACHE_TTL_SECONDS:-3600}
       - ARNS_CACHE_MAX_KEYS=${ARNS_CACHE_MAX_KEYS:-10000}
       - ARNS_CACHE_TYPE=${ARNS_CACHE_TYPE:-redis}
+      - ARNS_NAMES_CACHE_TTL_SECONDS=${ARNS_NAMES_CACHE_TTL_SECONDS:-3600}
```

Comment:
We should strip this default out of `docker-compose.yaml`. The default `config.ts` will handle a missing value. If you notice other obvious cases where we're doing that, feel free to correct them too, but we don't need to get them all in this PR.

Reply:
[`02cfccf` (#238)](https://github.com/ar-io/ar-io-node/pull/238/commits/02cfccf790fb875be4a9837ac282d15583721d83)